tools/pipelines/build-client.yml

```diff
@@ -60,6 +60,8 @@ trigger:
     - release/*
   paths:
     include:
+    # Trigger on any edits to files in the root
+    - "*.*"
     - .prettierignore
     - biome.json
     - biome.jsonc
```

Comment:
Why do we need to list `biome.json` and other root files directly, too?

Reply:
Hmmm, looks like a bad merge resolution on my part. One moment...

Reply:
Root file entries are now removed, the list is alphabetized and normalized as was done in the original reverted change (#22133).